lib/alarm.js

```diff
@@ -29,27 +29,29 @@ var util = require('util'),
 
 function raise(alarm, context, message) {
     var state = alarms[alarm];
-    context = (process.domain && process.domain.context) || {};
+    var contextTemp = (process.domain && process.domain.context) || {};
```

Comment:
Could you elaborate on the usage of contextTemp and messageTemp instead of context and message here?